SourceX/discord/discord.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef DISCORD
+
+#include <string>
+
+#define DISCORD_DIABLO_APPID 496571953147150354
+#define DISCORD_DEVIX_APPID 795760213524742205
+
+namespace discord {
+class Core;
+}
+
+namespace dvl {
+
+class DiscordManager {
+public:
+	DiscordManager();
+
+	void updateGame(int newNumPlayers, int newPlayerLevel, int newDungeonType, int newDungeonLevel);
+	void startGame(int difficulty, int maxPlayers, int plyrClass, const char *charName);
+
+	void setInMenuActivity();
+
+private:
+	std::string getPlayerInfoString();
+	std::string getLocationString();
+	std::string getDetailString();
+	std::string getStateString();
+
+private:
+	discord::Core *core = nullptr;
+
+	int numPlayers = 0;

```

Comment:
Please name private member fields like this (Google style):

```c++
int num_players_ = 0;
int max_players_ = 0;
...
```

Reply:
Where are these styles outlined for this project?

Reply:
Nowhere yet, there is no consistent style guide or a definitive clang-format

Some other classes are written this way, e.g. https://github.com/diasurgical/devilutionX/blob/master/SourceX/controls/devices/game_controller.h

Reply:
I'd like to hear from the other devs what style you guys would prefer as a group as I am personally not a fan of prefix/postfix underscores. Classes in [here](https://github.com/diasurgical/devilutionX/tree/master/SourceX/dvlnet) have a different style. @AJenbo @qndel 

I'll at least switch the variables to snake case naming which is more consistent between both examples.